controllers/release/adapter.go

```diff
@@ -720,13 +720,16 @@ func (a *adapter) getCollectorsPipelineRunBuilder(pipelineType, namespace, revis
 
 // createManagedCollectorsPipelineRun creates a PipelineRun to run the collectors Pipeline for collectors in the ReleasePlanAdmission.
 func (a *adapter) createManagedCollectorsPipelineRun(releasePlanAdmission *v1alpha1.ReleasePlanAdmission) (*tektonv1.PipelineRun, error) {
+	url, err := releasePlanAdmission.Spec.Pipeline.PipelineRef.GetUrl()
+	if err != nil {
+		url = "https://github.com/konflux-ci/release-service-catalog.git"
```

Comment:
I don´t like to see this set on the code. Shouldn't the ReleaseServiceConfig be used to set this?  @johnbieren  .

Reply:
I agree it isn't nice to hardcode this, but the argument can be made that it is already hardcoded today. I don't remember why though. Maybe @davidmogar can provide context as he is the author of https://github.com/konflux-ci/release-service/commit/6af748dbf11d8170ca78f3e40f607b52c1fc0683 which originally added it

Reply:
There no actual reason apart from that url being the only source for the collectors pipeline.

This change is mostly to enable testing, so using the ReleaseServiceConfig would make things more difficult than allow using the url of the RP/RPA. With the url being referenced twice now, I would add a constant.